### ISSUE TITLE
Registration returns low resolution results after update from version 1.0.0rc12 to 1.1.0

### ISSUE THREAD

**Issue by iMauricee** [MathOnco/valis#171]
Hello,

I am encountering an issue after upgrading from version 1.0.0rc12 to 1.1.0. Previously, my registration process worked as expected with the following code:
```python
registrar = registration.Valis(slide_src_dir, results_dir)
rigid_registrar, non_rigid_registrar, error_df = registrar.register()
registrar.register_micro(max_non_rigid_registration_dim_px=2000)
registrar.warp_and_save_slides(registered_slide_dst_dir, crop="overlap")
```
Since the update, using the same code, I now receive low-resolution results instead of the expected high resolution. When I separately attempt to open the respective slide file (.mrxs) using the following code:
```python
reader_cls = slide_io.get_slide_reader(slide_src_f, series=series)
reader = reader_cls(slide_src_f, series=series, use_openslide=True)
reader.create_metadata
meta = reader.metadata.slide_dimensions
```
The meta.slide_dimensions only shows a single low-resolution value (`[[616, 1436]]`) instead of the significantly larger original slide dimensions. However, when I open the same file using OpenSlide directly, I can retrieve all of the existing dimensions as expected:
```python
slide = openslide.OpenSlide(slide_src_f)
slide.level_dimensions
```
This returns: `((78753, 183759), ..., (615, 1435), (307, 717))`

Could you please help me understand why the dimensions are read incorrectly after the update/where I am making a mistake? Any guidance or suggestions would be greatly appreciated.

Thank you!

**Comment by cdgatenbee** [comment]
Hi @iMauricee,
Thanks for bringing this to my attention. I'm not entirely sure what is happening here, but could you try to the following to see if vips is finding openslide?
```
    import pyvips # python wrapper for vips
    from valis import slide_io

    # Check if vips is using openslide
    vips_img = pyvips.Image.new_from_file(mrxs_f, autocrop=True) # `autocrop` crops to region around the tissue(s), instead of the entire slide, which is often mostly empty. Setting `autocrop=False` will read the entire slide
    print(vips_img) # Should print shape and type of highest resolution level

    # Check if valis is extracting the correct metadata
    valis_vips_slide_reader = slide_io.VipsSlideReader(mrxs_f)
    print(valis_vips_slide_reader.metadata.slide_dimensions)
```

When I run the above on a local mrxs file (circa 2016) with vips 8.15 and openslide 4.0, the print commands produce the following:

```
>>> print(vips_img)
<pyvips.Image 103072x202595 uchar, 4 bands, srgb>

>>>  print(valis_vips_slide_reader.metadata.slide_dimensions)
array([[103072, 202595],
       [ 51536, 101298],
       [ 25768,  50649],
       [ 12884,  25325],
       [  6442,  12662],
       [  3221,   6331],
       [  1610,   3165],
       [   805,   1583],
       [   403,    792],
       [   202,    396]])
 
```

Please try the above and let me know what you get.

Best,
-Chandler

**Comment by iMauricee** [comment]
Hey @cdgatenbee, 

Thank you for your prompt response!
I ran the code with openslide 4.0 and vips 8.16 as well as 8.15 on my `.mrxs` file and both times received the following output:  
```
Vips: <pyvips.Image 616x1436 uchar, 3 bands, srgb>
VipsSlideReader: [[ 616 1436]]
[[ 616 1436]]
```
Additionally, I tested loading an `ome.tiff` generated from a previous computation using the old version, as well as a standard `.tiff`. Both of these provided the expected results, consistent with your example.

Thanks you for your support. 

Kind regards, 
Moritz
